hooks/pre/30-upgrade.rb

```diff
@@ -1,3 +1,13 @@
+def stop_services
+  Kafo::Helpers.execute('katello-service stop')
+end
+
+def start_mongo
+  status = Kafo::Helpers.execute('service-wait mongod status')
+  Kafo::Helpers.execute('service-wait mongod start') unless status
```

Comment:
I have seen this on slower VMs for EL7 because its more of a systemd issue. When we start mongod with upstart, it will just return thinking it has started mongo successfully, and then we use service-wait to poll for a condition before we mark it as 'truly' started. In the systemd case, it actually waits for the service to be entirely started and has a default 90 second timeout (which is the part I have run into on slower machines, https://bugzilla.redhat.com/show_bug.cgi?id=1040573)